terraform/modules/jenkins/network.tf

```diff
@@ -11,6 +11,11 @@ resource "aws_vpc" "main" {
   )
 }
 
+# Bring default security group under Terraform management and remove all rules
+resource "aws_default_security_group" "default" {
+  vpc_id = aws_vpc.main.id
```

Comment:
Does this security group need to be attached to a VPC at all? It sounds like it's optional: https://www.terraform.io/docs/providers/aws/r/default_security_group.html

Reply:
Thanks for explaining! I thought this was the default security group for the default VPC, but it's actually the default security group for our _custom_ VPC, so it does make sense to keep it attached to that VPC even though we've removed all the rules. :+1: